lib/plugins/aws/package/compile/events/apiGateway/lib/hack/updateStage.js

```diff
@@ -5,6 +5,8 @@
 const _ = require('lodash');
 const BbPromise = require('bluebird');
 
+const isRestApiId = RegExp.prototype.test.bind(/^[a-z0-9]{10}$/);
```

Comment:
Is this regex taken from AWS docs? Are we sure that the length should be 10?

Reply:
Good question. I was searching for some reference but was unable to find any.

It's just based on assumption (all ids I seen match that). Anyway I relaxed that